eclipse-sw360/sw360
# feat(rest) : Rest Api for ECC page

[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.
> * Which issue is this pull request belonging to and how is it solving it? (*Refer to issue here*)
> * Did you add or update any new dependencies that are required for your change?

Issue: 
#2036 

### Suggest Reviewer
> You can suggest reviewers here with an @mention.

### How To Test?
> How should these changes be tested by the reviewer?
> Have you implemented any additional tests?

http://localhost:8080/resource/api/ecc?page=0&page_entries=3&sort=name%2Cdesc

Sample response : 

{
    "_embedded": {
        "sw360:jSONObjects": [
            {
                "assessor_contact_person": "",
                "name": "compfeb2 (50)",
                "DT_RowId": "d29b5995f30146e5ad30b84a35f42112",
                "id": "d29b5995f30146e5ad30b84a35f42112",
                "version": "50",
                "assessor_dept": "",
                "status": "OPEN",
                "group": "",
                "assessment_date": ""
            },
            {
                "assessor_contact_person": "",
                "name": "Test Compo 2023 1 (22.23)",
                "DT_RowId": "e06f3a093ca1453085fd27c0017e6e2c",
                "id": "e06f3a093ca1453085fd27c0017e6e2c",
                "version": "22.23",
                "assessor_dept": "",
                "status": "OPEN",
                "group": "",
                "assessment_date": ""
            },
            {
                "assessor_contact_person": "eccadmin@sw360.org",
                "name": "Component5.19 (1.1)",
                "DT_RowId": "2e7f86c8236e4dd299328cbe3f7f7ae0",
                "id": "2e7f86c8236e4dd299328cbe3f7f7ae0",
                "version": "1.1",
                "assessor_dept": "DEPARTMENT",
                "status": "REJECTED",
                "group": "",
                "assessment_date": "2023-02-03"
            }
        ]
    },
    "_links": {
        "first": {
            "href": "http://localhost:8080/resource/api/ecc?page=0&page_entries=3"
        },
        "previous": {
            "href": "http://localhost:8080/resource/api/ecc?page=0&page_entries=3"
        },
        "last": {
            "href": "http://localhost:8080/resource/api/ecc?page=0&page_entries=3"
        },
        "curies": [
            {
                "href": "http://localhost:8080/resource/docs/{rel}.html",
                "name": "sw360",
                "templated": true
            }
        ]
    },
    "page": {
        "size": 3,
        "totalElements": 3,
        "totalPages": 1,
        "number": 1
    }
}

### Checklist
Must:
